frontend/views/containers/group-settings/GroupLeaveModal.vue

```diff
@@ -108,9 +108,9 @@ export default ({
       if (this.$v.form.$invalid) { return }
       try {
         const groupContractID = this.currentGroupId
-        await sbp('gi.actions/identity/leaveGroup', {
-          contractID: this.ourIdentityContractId,
-          data: { groupContractID }
+        await sbp('gi.actions/group/removeMember', {
+          contractID: groupContractID,
+          data: {}
```

Comment:
Curious, why was this change made? It seems like a significant logic change that is unrelated to the other thematic changes of this PR

Reply:
gi.actions/group/removeMember will eventually call `leaveGroup` as contracts are kept in sync. It could have been that this was breaking something too, but I don't remember.